.pre-commit-hooks.yaml

```
@@ -4,4 +4,12 @@
   language: haskell
   types:
     - nix
-  description: The official (but not yet stable) formatter for Nix code
+  description: &desc The official (but not yet stable) formatter for Nix code
+
+- id: &id-nix nixfmt-nix
+  name: *id-nix
+  entry: nix run github:NixOS/nixfmt
```

This will track our `main` branch, which could cause issues for users. How about the version in nixpkgs?

```suggestion
  entry: nix --extra-experimental-features "flakes nix-command" run nixpkgs#nixfmt-rfc-style
```

(not tested, please verify this works!)